src/main/kotlin/com/ampnet/reportserviceeth/blockchain/BlockchainServiceImpl.kt

```diff
@@ -26,6 +35,8 @@ class BlockchainServiceImpl : BlockchainService {
 
     override fun getWhitelistedAddress(issuer: String): List<String> {
         logger.debug { "Get whitelisted accounts for issuer: $issuer" }
-        TODO("Not yet implemented")
+        val contract = IIssuer.load(issuer, web3j, credentials, DefaultGasProvider())
```

Comment:
There is no need to load `Credenitals` because these are just public get calls without signing any transaction. 
```suggestion
        val transactionManager = ReadonlyTransactionManager(web3j, "")
        val contract = IIssuer.load(issuer, web3j, transactionManager, DefaultGasProvider())
```

See: https://github.com/web3j/web3j/issues/385